auraloss/freq.py

```diff
@@ -604,3 +604,175 @@ def forward(self, input: torch.Tensor, target: torch.Tensor):
             return loss
         elif self.output == "full":
             return loss, sum_loss, diff_loss
+
+class ModifiedSumAndDifferenceSTFTLoss(torch.nn.Module):
```

Comment:
Instead of creating a new class for this we should add parameters to `SumAndDifferenceSTFTLoss` in order to support this behavior. It seems like the major modification is that application of the pre-emphasis filter.